src/patients/patient-slice.ts

```diff
@@ -318,6 +351,80 @@ export const addDiagnosis = (
   }
 }
 
+function validatePhoneNumber(phoneNumber: PhoneNumber) {
```

Comment:
I think this function should also check the validity of the phone number. Then, wherever phone numbers are being validated, then this function should also be used. 

Type should also be a required field. 


(this is true for phone number, email, and address.